File: nowcoder/BC50_计算单位阶跃函数/BC50.go
package main
import "fmt"
func main() {
    for {
        var a int
        _, err := fmt.Scanf("%d",&a)
        if err != nil{
            break
        }
        if a >0 {
            fmt.Println("1")
        } else if a == 0 {
            fmt.Println("0.5")
        } else {
            fmt.Println("0")
        }
    }
     
     
}

File: nowcoder/BC50_计算单位阶跃函数/BC50.c
#include<stdio.h>
int main()
{
    int t;
    while(scanf("%d",&t)!=EOF)
    {
        if(t > 0)
            printf("1\n");
        else if(t ==0)
            printf("0.5\n");
        else
            printf("0\n");
        getchar();
    }
      
      
    return 0;
}
